pkg/antctl/transform/networkpolicy/response_test.go

```diff
@@ -182,7 +182,7 @@ func TestEvaluationResponseTransform(t *testing.T) {
 	assert.Equal(t, []string{"NAME", "NAMESPACE", "POLICY-TYPE", "RULE-INDEX", "DIRECTION", "ACTION"}, test.GetTableHeader())
 	assert.False(t, test.SortRows())
 	assert.Equal(t, []string{"", "", "", "", "", ""}, test.GetTableRow(32))
-	testDropAction := crdv1beta1.RuleActionDrop
+	testDropAction, testDefaultAction := crdv1beta1.RuleActionDrop, crdv1beta1.RuleActionAllow
```

Comment:
why is the name of the variable `testDefaultAction` and not `testAllowAction`? which default behavior are we referring to?

Reply:
I was referring to `defaultAction = secv1beta1.RuleActionAllow` assigned to K8s NP but I've changed to `testAllowAction`.